src/fides/api/api/v1/endpoints/dataset_config_endpoints.py

```diff
@@ -131,7 +138,7 @@ def validate_dataset(
     response_model=BulkPutDataset,
 )
 def put_dataset_configs(
-    dataset_pairs: Annotated[List[DatasetConfigCtlDataset], Field(max_length=50)],  # type: ignore
+    dataset_pairs: Annotated[List[DatasetConfigCtlDataset], Field(max_length=1000)],  # type: ignore
```

Comment:
Nit: might consider sharing a const between this and page size - even though this is a temp feature it makes it clear they are inter-dependent